src/spdx_tools/spdx3/writer/console/ai/ai_package_writer.py

```diff
@@ -0,0 +1,31 @@
+#  SPDX-FileCopyrightText: 2023 spdx contributors
+#
+#  SPDX-License-Identifier: Apache-2.0
+from typing import TextIO
+
+from spdx_tools.spdx3.model.ai import AIPackage
+from spdx_tools.spdx3.writer.console.artifact_writer import write_artifact_properties
+from spdx_tools.spdx3.writer.console.console import write_dict, write_value
+from spdx_tools.spdx3.writer.console.software.package_writer import write_package
+
+
+def write_ai_package(ai_package: AIPackage, text_output: TextIO):
+    text_output.write("## AI Package\n")
+    write_package(ai_package, text_output, False)
+    write_artifact_properties(ai_package, text_output)
```

Comment:
this method is already called in `write_package`, no need to duplicate this.

Reply:
good catch :)

Reply:
fixed